test/depositERC20.js

```diff
@@ -35,6 +35,67 @@ contract('Dead', (accounts) => {
       assert(deadFinalBalance.eq(ownerInitialBalance),
         `${accountabilityFail}. the DMS' final balance was not properly incremented`);
     });
+
+    it('should not allow a deposit larger than the sender\'s balance', async () => {
+      const dead = await Dead.deployed();
+      const token = await Token.deployed();
+      const errMsg = 'the owner was able to deposit with an amount greater than their initial balance';
+
+      //Record the initial balances of both accounts
+      const ownerInitialBalance = await token.balanceOf.call(owner);
+      const dmsInitialBalance = await token.balanceOf.call(dead.address);
+
+      //Attempt to deposit more tokens to the DMS than allowed
+      const attemptDepositAmount = ownerInitialBalance + 1;
+
+      try {
+        await depositTokens(owner, attemptDepositAmount, dead.address);
+        //The deposit was successful
+        assert(false, errMsg);
```

Comment:
Move this assert to after (outside) the catch block, and put a `return` at the end (inside) of the catch block.

Your code is fine, but better to not hit this catch block for an error we don't actually expect. What I mean by that is, we are handling the case in the catch here where `depositTokens` fails. We do not want to actually handle at all the case where it succeeds, we just want to blow up. Again, your code works, but cleaner to not catch the blow up at all, even if it will continue to blow up in the catch.